### DISCUSSION TITLE
Rarbg?

### DISCUSSION THREAD

**Discussion by Desertwolf89** [LivingWithHippos/unchained-android#258] [Q&A]
Was wondering if anyone can make a plugin for rarbg. Tried to make one and failed so I gave up

**Comment by LivingWithHippos** [comment]
I think I tried too but they use some kind of protection so you can't just scrape the data

**Reply by Desertwolf89** [reply]
Thank you for the response. I appreciate it. One of my favorite sites. I have no choice but to manually look for the magnets :)
